scripts/quests/jeuno/The_Gobbiebag_Part_I.lua

```diff
@@ -0,0 +1,131 @@
+-----------------------------------
+-- The Gobbiebag Part I
+-----------------------------------
+--  Log ID: 3, Quest ID: 27
+--  NPC: Bluffnix:  !pos -43 6 -115 245
+-----------------------------------
+require("scripts/globals/npc_util")
+require("scripts/globals/quests")
+require("scripts/globals/status")
+require("scripts/globals/titles")
+require("scripts/globals/utils")
+require("scripts/globals/items")
+require("scripts/globals/interaction/quest")
+-----------------------------------
+local lowerJeunoID = require("scripts/zones/Lower_Jeuno/IDs")
+-----------------------------------
+
+local GobbieQuest = {}
+
+setmetatable(GobbieQuest, {__index = Quest} )
+GobbieQuest.__index = GobbieQuest
+
+function GobbieQuest:new(params)
```

Comment:
I feel like the better place to store the base class would be in a helpers.lua file.  It feels a bit awkward for later quests to be calling in the initial quest script.

Reply:
Yeah... to be honest I have been playing around with this for a while before I finally got it to work properly. Agree with you on the awkwardness factor, I was hoping you guys would tell me a good place to put it :-) Sounds good, l will move and make sure everything works as expected.